talk/owt/sdk/ic/selfiesegmentationmodel.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) <2021> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OWT_IC_SELFIESEGMENTATIONMODEL_H_
+#define OWT_IC_SELFIESEGMENTATIONMODEL_H_
+
+#include <string>
+#include <vector>
+
+#include "inference_engine.hpp"
+#include "opencv2/core/mat.hpp"
+
+namespace owt {
+namespace ic {
+
+class SelfieSegmentationModel {
+ public:
+  explicit SelfieSegmentationModel(InferenceEngine::Core& core);
+
+  bool LoadModel(const std::string& xmlPath, const std::string& device = "CPU");
+  bool IsLoaded() const;
+
+  cv::Mat predict(const cv::Mat& frame);
+  void predictAsync(const cv::Mat& frame);
+  cv::Mat waitForFinished();
+
+ private:
+  InferenceEngine::Core &core;
```

Comment:
nit: all class data members needs to be named with _ suffixed, so it should be input_name_, output_name_ , etc.

Reply:
OK. And is there any linter that can automatically do this?